flink-libraries/flink-table/src/test/scala/org/apache/flink/table/utils/TableTestBase.scala

```diff
@@ -151,21 +152,31 @@ case class BatchTableTestUtil() extends TableTestUtil {
     verifyTable(tEnv.sql(query), expected)
   }
 
-  def verifyTable(resultTable: Table, expected: String): Unit = {
+  def verifyTable(resultTable: Table, expected: String, tableEnv: BatchTableEnv): Unit = {
```

Comment:
Because we should verify JavaTableEnv, I don't remove the changes. Is there any good way to do this?